build-system/git.js

```diff
@@ -24,55 +24,45 @@ const {getStdout} = require('./exec');
 
 const commitLogMaxCount = 100;
 
-/**
- * Returns the merge base of the current branch off of master when running on
- * a local workspace.
- * @return {string}
- */
-exports.gitMergeBaseLocalMaster = function() {
-  return getStdout('git merge-base master HEAD').trim();
-};
-
 /**
  * Returns the merge base of the current branch off of master, regardless of
  * the running environment.
  * @return {string}
  */
 exports.gitMergeBaseMaster = function() {
   if (process.env.TRAVIS) {
-    const commitRange = process.env.TRAVIS_COMMIT_RANGE.split('...');
-    return getStdout(`git merge-base ${commitRange[0]} ${commitRange[1]}`)
-        .trim();
+    return getStdout(
+        `git merge-base master ${process.env.TRAVIS_PULL_REQUEST_SHA}`).trim();
   }
-  return exports.gitMergeBaseLocalMaster();
+  return gitMergeBaseLocalMaster();
 };
 
 /**
  * Returns the `master` parent of the merge commit (current HEAD) on Travis.
  * @return {string}
  */
 exports.gitTravisMasterBaseline = function() {
-  return process.env.TRAVIS_COMMIT_RANGE.split('...')[0];
+  return getStdout('git rev-parse origin/master').trim();
 };
 
 /**
- * Returns the list of files changed on the local branch relative to the branch
- * point off of master, one on each line.
+ * Returns the list of files changed relative to the branch point off of master,
+ * one on each line.
  * @return {!Array<string>}
  */
 exports.gitDiffNameOnlyMaster = function() {
-  const branchPoint = exports.gitMergeBaseLocalMaster();
-  return getStdout(`git diff --name-only ${branchPoint}`).trim().split('\n');
+  const masterBaseline = gitMasterBaseline();
```

Comment:
This was changed to use the function that accounts for local vs. Travis differences. ✓